nifi-toolkit/nifi-toolkit-cli/src/main/java/org/apache/nifi/toolkit/cli/impl/command/nifi/pg/PGImport.java

```diff
@@ -85,6 +93,35 @@ public StringResult doExecute(final NiFiClient client, final Properties properti
 
         final boolean posXExists = StringUtils.isNotBlank(posXStr);
         final boolean posYExists = StringUtils.isNotBlank(posYStr);
+        final File input = new File(inputSource);
+
+        if (StringUtils.isBlank(inputSource)) {
+            if (StringUtils.isBlank(bucketId)) {
+                throw new IllegalArgumentException("Input path is not specified so Bucket ID must be specified");
+            }
+            if (StringUtils.isBlank(flowId)) {
+                throw new IllegalArgumentException("Input path is not specified so Flow ID must be specified");
+            }
+            if (StringUtils.isBlank(flowVersion)) {
+                throw new IllegalArgumentException("Input path is not specified so Flow Version must be specified");
+            }
+        } else {
+            if (!input.exists() || !input.isFile() || !input.canRead()) {
+                throw new IllegalArgumentException("Specified input is not a local readable file: " + inputSource);
+            }
+            if (!StringUtils.isBlank(bucketId)) {
+                throw new IllegalArgumentException("Input path is specified so Bucket ID should not be specified");
+            }
+            if (!StringUtils.isBlank(flowId)) {
+                throw new IllegalArgumentException("Input path is specified so Flow ID should not be specified");
+            }
+            if (!StringUtils.isBlank(flowVersion)) {
+                throw new IllegalArgumentException("Input path is specified so Flow Version should not be specified");
+            }
+            if (!StringUtils.isBlank(flowBranch)) {
+                throw new IllegalArgumentException("Input path is specified so Flow Branch should not be specified");
+            }
```

Comment:
Instead of `!StringUtils.isBlank` use Apache Commons `StringUtils isNotBlank` as you did on line 188. This may be a preference but I believe the use of this conveys a clearer intention.